picdar-export/src/main/scala/com.gu.mediaservice.picdarexport/lib/usage/ResponseParser.scala

```diff
@@ -0,0 +1,17 @@
+package com.gu.mediaservice.picdarexport.lib.usage
+
+
+object ResponseParser {
```

Comment:
Could be useful to dump a sample response in a comment here for future reference?